### ISSUE TITLE
QA Ph3_Accessibility: Schedule - Missing alternative text and empty links 

### ISSUE THREAD

**Issue by brittasinnott** [GoogleChrome/ioweb2016#636]
There are a number of errors on the Schedule pages when using the WAVE extension for Chrome, set to filter WCAG 2.0 AA. The same errors persist across all Schedule pages. 
-4 cases of missing alternative text
-3 cases of empty headers 
-2 cases of empty links 

Steps: 
1. Go to https://io-webapp-staging.appspot.com/io2016/
2. Select Schedule
3. Enable the WAVE extension
4. View Style errors with the filter set to WCAG 2.0 AA 
5. Note 9 errors 

Chrome 48 Mac OS X 10.11

![gio_636](https://cloud.githubusercontent.com/assets/15128124/14444608/73611e5c-fffb-11e5-9d5d-7c8e81c65948.png)
